src/main/java/com/wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(unique = true)
+    @NotEmpty
+    private String username;
+
+    @NotEmpty
+    private String name;
+
+    @Past
+    @NotEmpty
```

Comment:
@NotEmpty only supports the type CharSequence, Collection, Map, or Array, so LocalDate is not included in any of these types. When trying to use the model, it will break. I would use for this data type @NotNull for the null check.

I would also add the nullable false in the @Column

```suggestion
    @NotNull
    @Column(nullable = false, unique = true)
```